LayoutTests/http/tests/dom/focus-in-cross-origin-iframe.html

```diff
@@ -0,0 +1,33 @@
+<script>
+
+if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+}
+
+window.addEventListener("message", (event) => {
+	document.getElementById("result").innerText = event.data;
```

Comment:
Tab.

Reply:
I think it's especially appropriate to have tabs in this test